packages/@lwc/template-compiler/src/codegen/index.ts

```diff
@@ -402,6 +406,7 @@ function transform(codeGen: CodeGen): t.Expression {
         const innerHTML = element.directives.find(isInnerHTMLDirective);
         const forKey = element.directives.find(isKeyDirective);
         const dom = element.directives.find(isDomDirective);
+        const addSanitizationHook: boolean = isCustomRendererHookRequired(element, codeGen.state);
```

Comment:
No need to explicitly mention the return type. Could this variable declaration be renamed to something more explicit like: `useCustomRenderer`.

```suggestion
        const useCustomRenderer = isCustomRendererHookRequired(element, codeGen.state);
```